apps/pie-storybook/stories/pie-checkbox.stories.ts

```diff
@@ -33,9 +33,6 @@ const checkboxStoryMeta: CheckboxStoryMeta = {
         value: {
             description: 'The value of the checkbox (used as a key/value pair in HTML forms with `name`).',
             control: 'text',
-            defaultValue: {
-                summary: defaultArgs.value,
```

Comment:
we don't need this anymore?

Reply:
Yes, it is not needed because we set a default value "on" inside the component. So for users there is no default value. It mimics the behaviour of a native checkbox